presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -878,6 +878,11 @@ public void testStringFunctions()
         assertQuery("SELECT substr(comment, 1, 10), length(comment), ltrim(comment) FROM orders");
         assertQuery("SELECT substr(comment, 1, 10), length(comment), rtrim(comment) FROM orders");
 
+        // Substring, length, trim. Alias to Substr
```

Comment:
Lets move these tests to presto-native-tests module. So we will only need to add the teradata module dependencies there instead of presto-native-execution.